.github/workflows/ci.yml

```
@@ -1,21 +1,67 @@
 
 permissions:
   contents: read
 
 jobs:
-  build:
-    name: Lint and Build
+  # Native App Checks (Release 1.0 Focus)
+  native-checks:
+    name: Native App (Typecheck & Lint)
     runs-on: ubuntu-latest
+    if: inputs.target != 'web' # Always run unless explicitly skipped (default behavior)
```

actionlint (1.7.10)

property "target" is not defined in object type {}

(expression)